app/components/RegisterSingleTree/SingleTreeOverview.js

```diff
@@ -12,7 +12,6 @@ import {
   Platform,
   TextInput,
   Dimensions,
-  Alert,
```

Comment:
This leads to an error #151 as Alert is still used below. This can be avoided by using the lint checks and fix all the issues as proposed in #149!

Reply:
Fixed with linting error fix commit